source/stylesheets/_headings.scss

```diff
@@ -1,15 +1,17 @@
-h2.underlined {
-  color: rgba(101, 87, 87, 0.67);
-  margin-bottom: 1em;
+.underlined-heading {
+  color: rgb(183, 183, 183);
```

Comment:
Yeah we can do that! I don't know what the rules are with SCSS/SASS on using variables in the way the CSS pipeline is setup. It should be possible but I think you need to require the variables or something like that. Either way, go for it—sounds like a good idea to me!